Package.swift

```diff
@@ -13,6 +13,7 @@ let package = Package(
 		.library(name: "SwiftLexicon", targets: ["SwiftLexicon"]),
 		.library(name: "SwiftStandAlone", targets: ["SwiftStandAlone"]),
 		.library(name: "KotlinStandAlone", targets: ["KotlinStandAlone"]),
+        .library(name: "TypeScriptStandAlone", targets: ["TypeScriptStandAlone"]),
```

Comment:
Tab vs spaces. Could you update Xcode to use tabs. See [this comment](https://github.com/thousandyears/Lexicon/pull/1#issuecomment-1109640856).